### PR TITLE
Replace hard-coded value for app_version with actual version in CaseSessionDataHelper

### PR DESCRIPTION
**Pull request by gherceg**
## Technical Summary

A pending formplayer change, https://github.com/dimagi/formplayer/pull/1075, will make use of the `appVersion` property in session data for a new sms forms session request. **Currently nothing in formplayer makes use of that property which is why it is safe to change what this value represents.** 
### Safety story

A trivial change. A previously hard-coded value is now replaced by `app.version`. 
### Automated test coverage

I don't think it is worth adding automated test coverage for this data class.
### QA Plan

Will not run through QA before merged to master, but will run through QA as part of the formplayer work.

### Rollback instructions

Rolling this back will break the formplayer change referenced above once that is deployed. 

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by gherceg** [comment]
> I don't think it is worth adding automated test coverage for this data class.

Ha totally missed the start session tests! Thanks for adding those @mkangia

File: corehq/apps/cloudcare/touchforms_api.py [hunk 1]
```diff
@@ -18,7 +18,6 @@ def get_session_data(self, device_id=CLOUDCARE_DEVICE_ID):
         """
         session_data = {
             'device_id': device_id,
-            'app_version': '2.0',
```
Comment (millerdev) [inline corehq/apps/cloudcare/touchforms_api.py:21]: This looks like [`APP_V2`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/const.py#L2) as in [`Application.application_version`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/models.py#L4191), which has not seen a new version in years. Is this PR changing the meaning of `session_data["app_version"]` to now be [`Application.version`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/models.py#L4112) (which is an integer, not a string)?

Does anything in formplayer depend on the old meaning and maybe needs to be updated?

Comment (gherceg) [reply corehq/apps/cloudcare/touchforms_api.py:21]: Yep this is changing the meaning. From the description "Currently nothing in formplayer makes use of that property which is why it is safe to change what this value represents", but I can make that more clear.

Comment (millerdev) [reply corehq/apps/cloudcare/touchforms_api.py:21]: I missed that in the PR description, so thanks for clarifying here. No need for further clarification.

**Review by millerdev — APPROVED** [review]

**Pull request merged**